db/schema.rb

```diff
@@ -64,14 +64,10 @@
   create_table "menu_items", force: :cascade do |t|
     t.string   "name"
     t.decimal  "price"
-    t.datetime "created_at",         null: false
-    t.datetime "updated_at",         null: false
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
     t.text     "description"
     t.string   "ingredients"
-    t.string   "image_file_name"
-    t.string   "image_content_type"
-    t.integer  "image_file_size"
-    t.datetime "image_updated_at"
   end
```

Comment:
Those attributes were associated with the paperclip gem.  Not needed now as we have moved to Cloudinary.